### PR TITLE
New DPA Parser implementation [SDESK-7423]

### PR DESCRIPTION
**Pull request by devketanpro**

### CODE REVIEW

File: tests/io/feed_parsers/dpanewsml_test.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            self.assertEqual(self.item.get("renditions"), {})
+
+    def test_word_count(self):
+        with self.app.app_context():
+            self.assertEqual(self.item.get("word_count"), 226)
+
+    def test_body_html(self):
+        with self.app.app_context():
+            self.assertIsInstance(self.item.get("body_html"), type(""))
+            self.assertEqual(self.item.get("body_html"), "<header>  </header>")
```
Comment (petrjasek) [inline tests/io/feed_parsers/dpanewsml_test.py:88]: this seems wrong, the content should be like:
```
<p><span class="dateline">London <span class="credit">(dpa)</span> - </span>2019 gab es bittere Tränen in der Kurve, 2022 erst mächtig Wut auf englische Fans und eine Woche später ausgelassenen Jubel: Für Eintracht Frankfurt ist London in den vergangenen Jahren zu einem Standard-Reiseziel im europäischen Fußball-Wettbewerb geworden. Bevor es am Mittwochabend (21.00 Uhr/DAZN) bei Tottenham Hotspur um wichtige Punkte in der Champions League geht, wird sich die Reisegruppe um Torhüter Kevin Trapp und Routinier Makoto Hasebe bestimmt an die vergangenen London-Reisen erinnern.</p><p>Beim Topclub FC Chelsea wollte die damals von Adi Hütter trainierte Eintracht in der Saison 2018/19 ihre Erfolgsserie fortsetzen und nach Inter Mailand und Schachtjor Donezk auch die «Blues» aus der Europa League werfen. Nach zwei 1:1 ging es an der Stamford Bridge in die Verlängerung und ins Elfmeterschießen. Der inzwischen abgetretene Martin Hinteregger vergab vom Punkt, vergoss nach dem bitteren Aus Tränen und wurde anschließend in der Fankurve getröstet.</p><p>Sportlich weckt der 2:1-Erfolg im Halbfinal-Hinspiel bei West Ham United Ende April positive Erinnerungen - schließlich war er der Grundstein für den späteren Triumph in Sevilla. Doch eine Attacke von englischen Fans gegen zwei Journalisten des Hessischen Rundfunks trübte das Bild. Die Rundfunk-Reporter bekamen nach eigenen Angaben «mehrfach Faustschläge an den Hinterkopf, in den Nacken, in den Rücken». West Ham United machte die Täter später ausfindig. Eine Woche später gewann die Eintracht auch das Rückspiel und zog ins Endspiel ein.</p><p> </p>
```

**Review by petrjasek — COMMENTED** [review]
can you also parse keywords? those might be useful

**Comment by devketanpro** [comment]
> can you also parse keywords?

can you please let me know the tag which we have parsed as a keyword?

**Comment by petrjasek** [comment]
```
<keyword rank="1">Champions League</keyword>
<keyword rank="2">Frankfurt</keyword>
<keyword rank="3">London</keyword>
```

**Review by petrjasek — APPROVED** [review]

**Pull request merged**